.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: ruby
 rvm:
-    - 1.9.2
```

Comment:
Not that I mind, but are we now using a feature in `1.9.3` that isn't in `1.9.2`?